package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "colors",
     "description": "get colors in your node.js console",
-    "version": "1.4.44-liberty-2",
+    "version": "1.5.0",
```

Comment:
better use 1.4.5 in order to updated all

Reply:
The previous one is 1.4.44 (-liberty-2).  So it would have to be 1.4.45 if you think it needs to be a minor version.  Was just attempting to make it simple for people to avoid all of the 1.4.X code given the severity of this.